pkg/controller/atlasdatabaseuser/atlasdatabaseuser_controller.go

```diff
@@ -43,18 +44,24 @@ import (
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/workflow"
 )
 
+//nolint:stylecheck
+const FeatureOIDC = "FEATURE_PREVIEW_OIDC_DB_ACCESS"
```

Comment:
non-blocking thought: I wonder if we rather want to gather all features in a central `features.go` file rather than sprinkling them in controller logic implementation files 🤔

Reply:
Fixed :)